### PR TITLE
update unsupported view

### PR DESCRIPTION
**Pull request by pearl-truss**
This PR updates the content and styling of the unsupportedView driver component

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -114,6 +114,22 @@ case 'rtf':
...
 ```
 
+### Viewing local changes in a secondary repo
+
+If you are working on a feature branch and need to see changes introduced in that branch in another repo that using this library, here are the steps:
+
+1. Run `yarn build` in this react-file-viewer repo
+2. In the secondary repo, update `package.json` to point the trussworks/react-file-viewer declaration to your branch in the react-file-viewer repo:
+
+```"@trussworks/react-file-viewer": "git+https://github.com/trussworks react-file-viewer#your-branch-name"
```
Comment (ronaktruss) [inline README.md]: You'll need to add a newline for this to be displayed properly. (Ie. have the three distinct lines, two for the beginning and ending ``` and one for the code itself.)

**Review by ronaktruss — APPROVED** [review]
I wasn't able to run this locally (had some local environment issues with node), but the changes look fine.

**Comment by pearl-truss** [comment]
@ronaktruss thanks for the review. Do you think I should update this so that it doesn't update the UnsupportedView component for all other projects that are also using this repo? I was wondering if I should put these changes behind a prop

**Comment by ronaktruss** [comment]
@pearl-truss I was wondering something similar. I think this has merits as the default behavior. 

Do we know if there are other projects using this? (I didn't see any with a quick github search).

**Comment by pearl-truss** [comment]
> @pearl-truss I was wondering something similar. I think this has merits as the default behavior.
> 
> Do we know if there are other projects using this? (I didn't see any with a quick github search).

Not that I saw either but I just asked in #engineering on Truss slack to be sure

**Comment by pearl-truss** [comment]
> > @pearl-truss I was wondering something similar. I think this has merits as the default behavior.
> > Do we know if there are other projects using this? (I didn't see any with a quick github search).
> 
> Not that I saw either but I just asked in #engineering on Truss slack to be sure

@duncan-truss do you have an opinion on this?

**Pull request merged**